README.md

```diff
@@ -27,3 +27,11 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+I had the help of chat gpt with the conversion of making my program recursive instead of direct.
+
+A good invariant for this implementation would be, the array returned by fib(n) has the first n + 1 fib numbers from F(0) to F(n) and for any i >= 2 because of the base cases, each element satisfies F(i) = F(i-2) + F(i-1).
```

Comment:
Not following you here -- so to compute the $n$th Fibonacci number, it must already be there?

Reply:
The invariant is telling us that the Fibonacci numbers before F(n) are the correct FIbonacci numbers in order all the way up until F(n-1)

Reply:
So not the first n + 1 Fibonacci numbers?

Reply:
no because we are calculating the current fibonacci numbers which would just be the n term and we need to make sure that the fibonacci numbers that came before it are the correct ones making it n -1

Reply:
So then that's what your invariant should say.

Reply:
i fixed it in the readme file